std/dynamic_library.zig

```diff
@@ -119,7 +119,7 @@ pub const LinuxDynLib = struct {
             null,
             size,
             linux.PROT_READ | linux.PROT_EXEC,
-            linux.MAP_PRIVATE | linux.MAP_LOCKED,
+            linux.MAP_PRIVATE,
             fd,
             0,
         );
```

Comment:
We don't seem to check the result of this `mmap` call?

Reply:
Should it be added to this PR?